conf/application.conf

```diff
@@ -181,4 +182,8 @@ eori-integration {
   enrolment-url = ""
 }
 
+file-upload {
+  timeout = 10 seconds
```

Comment:
OBSERVATION: If we change the time unit, this may also need updating.

Reply:
This config value can be specified in whatever units you like (e.g. 0.5 minutes).  It will be automatically converted to seconds in FrontendAppConfig